controllers/databasecluster_controller.go

```diff
@@ -2844,6 +2844,20 @@ func (r *DatabaseClusterReconciler) updateObject(ctx context.Context, obj, oldOb
 	return r.Update(ctx, obj)
 }
 
+func (r *DatabaseClusterReconciler) parseIPSourceRanges(ranges []string) []string {
```

Comment:
Can we have tests for this function?
Also, what about using https://pkg.go.dev/net#ParseIP and https://pkg.go.dev/net#ParseCIDR?

Reply:
Updated